packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -171,14 +171,13 @@ $mdc-grid-list-tile-secondary-icon-size: 24px;
     font-size: 1rem;
     font-weight: #{map-get($mdc-typography-font-weight-values, medium)};
     line-height: 1rem;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
+@include mdc-typography-overflow-ellipsis;
```

Comment:
1- Add an empty line before this one with 0 spaces/indentation
2- Add an indentation of 4 spaces here